index.js

```diff
@@ -572,7 +578,9 @@ TChannelConnection.prototype.handleReqFrame = function (reqFrame) {
 
 	this.inPending++;
 	var opCallback = responseFrameBuilder(reqFrame, sendResponse);
-	var op = this.inOps[id] = new TChannelServerOp(this, handler, reqFrame, opCallback);
+	var op = this.inOps[id] = new TChannelServerOp(this, handler, reqFrame, {
+		
```

Comment:
looks like you're passing options (why this whitespace) then start, but the constructor takes start then options